torchci/scripts/deleteOldQueryLambdaVersions.mjs

```diff
@@ -0,0 +1,75 @@
+// Script to delete old versions of query lambdas
+
+import rockset from "@rockset/client";
+import { promises as fs } from "fs";
+
+const oneMonthAgo = new Date(Date.now() - 30 * 24 * 60 * 60 * 1000);
+
+async function readJSON(path) {
+  const rawData = await fs.readFile(path);
+  return JSON.parse(rawData);
+}
+
+const client = rockset.default(process.env.ROCKSET_API_KEY);
+
+const prodVersions = await readJSON("./rockset/prodVersions.json");
+Object.keys(prodVersions).forEach((workspace) => {
+  Object.entries(prodVersions[workspace]).forEach(
+    async ([queryName, jsonVersion]) => {
+      try {
+        const versions = await client.queryLambdas.listQueryLambdaVersions(
+          workspace,
+          queryName
+        );
+        const prodVersion = (
+          await client.queryLambdas.getQueryLambdaTagVersion(
+            workspace,
+            queryName,
+            "prod"
+          )
+        ).data.version.version;
+        const latest = (
+          await client.queryLambdas.getQueryLambdaTagVersion(
+            workspace,
+            queryName,
+            "latest"
+          )
+        ).data.version.version;
+
+        for (const versionInfo of versions.data) {
+          const version = versionInfo.version;
+          const lastExecutedAt = versionInfo.last_executed_at;
+          const createdAt = versionInfo.created_at;
+          if (
+            Date.parse(lastExecutedAt) > oneMonthAgo ||
```

Comment:
How does this treat versions that were never executed?

Reply:
It evaluates to false